api/question/schema.py

```diff
@@ -60,6 +61,7 @@ def resolve_questions(self, info):
 class UpdateQuestion(graphene.Mutation):
     class Arguments:
         question_id = graphene.Int(required=True)
+        question_title = graphene.String(required=True)
```

Comment:
I do not think the question title should be required in the update mutation since it seems you will be forcing the user to update the question_title every time they need to update something.

Reply:
@caveinn I assume when the question changes, one might want to change the title too. Also since the `question_title` field has been assigned a default value for the questions that are already existing, this is to give the front end team a chance to be able to edit the titles to what they see fit for each question.

Reply:
@caveinn I had miss understood your concern but it's now clear and it has been rectified.